docs/operations/images.md

```diff
@@ -1,167 +1,235 @@
 # Images
 
-Changing the image for an instance group
+The default images for Kops clusters are based on Debian 9 (Stretch). These images are built by the [kubernetes-sigs/image-builder](https://github.com/kubernetes-sigs/image-builder/tree/master/images/kube-deploy/imagebuilder) project for each Kubernetes release and are available for any other use under the `kope.io` account.
 
-You can choose a different AMI for an instance group.
+You can choose a different image for an instance group by editing it with `kops edit ig nodes`. You should see an `image` field in one of the following formats:
 
-If you `kops edit ig nodes`, you should see an `image` member of the spec.
+* `ami-abcdef` - specifies an AMI by id directly
+* `<owner>/<name>` specifies an AMI by its owner and name properties
```

Comment:
```suggestion
* `<account ID>/<name>` specifies an AMI by its owner's account ID and name properties
```

Reply:
The `help` command for `aws-cli` uses the "owner" word for the flag. I think a good compromise would be adding "account ID" in the description.
```
       --owners (list)
          Filters the images by the owner. Specify an  AWS  account  ID,  self
          (owner  is  the sender of the request), or an AWS owner alias (valid
          values are amazon | aws-marketplace |  microsoft  ).  Omitting  this
          option  returns  all  images  for which you have launch permissions,
          regardless of ownership.
```

Reply:
ah, good call. that sounds good to me 👍